vip_hci/preproc/recentering.py

```diff
@@ -343,6 +343,10 @@ def frame_center_satspots(array, xy, subi_size=19, sigfactor=6, shift=False,
         See the documentation of the ``vip_hci.preproc.frame_shift`` function.
     fit_type: str, optional {'gaus','moff'}
         Type of 2d fit to infer the centroid of the satellite spots.
+    filter_freq: tuple of 2 elements. 
+    If the first and second elements are bigger than 0, then the first element
```

Comment:
Replace with:
"If the first (resp. second) element of the tuple is larger than 0, a high-pass (resp. low-pass) filter is applied to the image, before fitting the satellite spots. The elements should correspond to the `fwhm_size` of the `frame_filter_highpass` and `frame_filter_lowpass` functions, respectively. If both elements are non-zero, both high-pass and low-pass filter of the image are applied, in that order. This can be useful to better isolate the signal from the satellite spots."